playground/vite.config.ts

```diff
@@ -58,7 +58,7 @@ export default defineConfig(({ mode }) => {
       //   ],
       // }),
       bundlesize({
-        limits: [{ name: 'assets/index-*', limit: '1600kB' }],
+        limits: [{ name: 'assets/index-*', limit: '1650kB' }],
```

Comment:
@Thunkar Was having problems just hitting the size limit here after adding a package to `foundation`. Not sure of the best way to solve it (since the package is needed) so bumped this - any advice appreciated!

Reply:
It's been bumped to 1680kb due to bb.js changes, would that work? 

It's ok as long as it's justified (and this is!), I just want to make sure it doesn't get completely out of control again!

Reply:
Shouldn't be an issue! When I merge master if I need to add a lot more to get it working I will get in touch, thank you!

Reply:
Had to bump from 1700 -> 1720 just now. Running CI to see if it works, but happy to take a look and find an alternative if you feel 1720 is pushing the limit!